vidarr-server/src/test/java/ca/on/oicr/gsi/vidarr/server/MainIntegrationTest.java

```diff
@@ -1008,10 +1212,11 @@ private ObjectNode getBcl2FastqUnloadFilter() {
     return unloadFilter;
   }
 
-  private long dateFromTime(String timeString) throws ParseException {
-    SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX");
-    df.setTimeZone(TimeZone.getTimeZone("EST"));
-    return df.parse(timeString).getTime();
+  private Instant dateFromTime(String timeString) throws ParseException {
+    return OffsetDateTime.parse(timeString).toInstant();
+    //    SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX");
```

Comment:
Old code commented out?

Reply:
Thanks! I've removed the commented out code.